src/components/augmint-ui/button/index.js

```diff
@@ -18,5 +19,16 @@ export default function Button(props) {
         _icon = <Icon name={icon} />;
     }
 
-    return React.createElement(elementType, props, children, content, _icon);
+    return (
+        <FeatureContext.Consumer>
+            {features => {
+                const dashboard = features.dashboard;
+                let _className = className;
+                if (dashboard) {
```

Comment:
fun fact: this could be a ternary one liner: 
```
const _className = dashboard ? _className += " dashboardColors" : _className;
```
... but perfectly fine as it is now!

Reply:
cool thx ;) i saw this somewhere in the codebase, i tryied it but eslint said many things so i just used the way i copied it over :D :(